app/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.iic.shopingo">
+    package="com.iic.shopingo" >
 
-  <application android:allowBackup="true"
-      android:label="@string/app_name"
-      android:icon="@mipmap/ic_launcher"
-      android:theme="@style/AppTheme">
+  <uses-permission android:name="android.permission.INTERNET" />
 
-  </application>
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:theme="@style/AppTheme" >
+        <activity
+            android:name=".ui.ContactDetailsActivity"
+            android:label="@string/title_activity_contact_details" >
+        </activity>
+        <activity
+            android:name=".ui.OnboardingActivity"
+            android:label="@string/title_activity_onboarding" >
+          <intent-filter>
+            <action android:name="android.intent.action.MAIN" />
```

Comment:
We said that the main activity was gonna be the home page, instead of the onboarding.